format/index.md

```diff
@@ -513,6 +513,39 @@ the following members:
   data represents a resource relationship.
 * Pagination links for the primary data (as described below).
 
+Value of each member of a `links` object can be either a string
+containing the link URL or a "link object", which can contain
+the following members:
+
+* `"href"` - the string containing the link URL or an array containing multiple strings containing the link URLs,
```

Comment:
Oh. Another exhausted ah ha moment. I forgot that all the `links` keys/relations we currently have defined _only make sense_ with a single URI. I've been thinking so much about hierarchical types and extensions that I keep thinking that we already have `collection` and `profile` keys. 

Can we just say that each key in `links` may accept a single link (given as a URI or object + `href`) or an array of such links, but then we specify that `self`, `related`, etc. only take one link? That way, the array form doesn't apply to them, but clients know that arrays may be there in the future (and even in 1.0 for `profile`, if #650 gets merged).